viewing_party/party.py

```diff
@@ -1,23 +1,194 @@
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
+def create_movie(movie_title, genre, rating):
+
+    movie= {}
+
+    if movie_title and genre and rating:
+    
+        movie["title"] = movie_title
+        movie["genre"] = genre
+        movie["rating"] = rating          
+        return movie
+
+    return None
+
+def add_to_watched(user_data, movie):
+
+    # Create a copy of user data to avoid modifying original
+    updated_user_data = {
+            "watched": user_data["watched"][:]}

```

Comment:
I see that you took into consideration that `user_data` was a mutable data type and decided to make a copy of the structure to avoid modifications of the original. I love that you are thinking about things like and applying your learnings from class! In this case, its actually fine to modify the original `user_data` since they ask you too in the requirements. Since the updated structure is returned and that return value is what is tested all your tests pass. If we were to test against the original `user_data` it would not.